modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -51,25 +49,6 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
 the changes from the codemod as a single isolated commit (separate from other changes) so you can
 roll back more easily if necessary.
 
-## Removals
-
-Removals are deletions from our codebase and you can no longer consume this component. We've either
-promoted or replaced a component or utility.
-
-### Menu (Preview)
-
-We've removed the `Menu` component in Preview. We recommend using the
-[Menu](/components/popups/menu/) in the Main package instead; it is a
-[compound component](/getting-started/for-developers/resources/compound-components/) and offers a
-more flexible API. Please refer to this
-[discussion](https://github.com/Workday/canvas-kit/discussions/2063) for instructions on how to
-migrate from the `Menu` in Preview to the `Menu` in Main.
-
-This change is **not** handled by the codemod due to the differences in API between the `Menu` in
-Preview and the `Menu` in Main.
-
-## Deprecations
-
```

Comment:
I don't think anything needs to go into the upgrade guide since we deprecated it in v8?

Reply:
Makes sense to me.

Removing the `## Deprecations` heading also makes sense since that section is empty, though this means we should also remove the [Deprecations line](https://github.com/Workday/canvas-kit/pull/2335/files#diff-f8fc2485022be76f6e0fffcbd06a87a52bd3d77ef9c70837860889f51b471b86R12) in the table of contents at the top.

Reply:
Are we not deprecating anything in v10?